docs/implementation/computation.rst

```diff
@@ -0,0 +1,187 @@
+===========
+Computation
+===========
+
+Fitting Bayesian statistical models of large metabolic networks poses
+distinctive computational problems. This document explains what the problems
+are and how Maud attempts to address them.
+
+Goal
+====
+
+Maud aims to fit Bayesian models of metabolic networks with about 100 internal
+reactions, each with its kinetics represented by the modular rate law and MWC
+formalism as described in the section on :doc:`Maud's statistical model
+<../theory/statistics>`, and around 20 experiments.
```

Comment:
Not sure if this should be in the computational docs. I think it would be more applicable as a milestone because the 20 experiments seems quite arbitrary and could expand to >100

Reply:
You're right, it's not really necessary to include specific numbers - I've rephrased.